src/modules/cmdpal/Microsoft.CmdPal.UI/Controls/ContentIcon.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using CommunityToolkit.WinUI;
+using Microsoft.CmdPal.UI.Deferred;
+using Microsoft.CmdPal.UI.ViewModels;
+using Microsoft.UI.Dispatching;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Windows.Foundation;
+
+namespace Microsoft.CmdPal.UI.Controls;
+
+/// <summary>
+/// A helper control which takes an <see cref="IconSource"/> and creates the corresponding <see cref="IconElement"/>.
+/// </summary>
+public partial class ContentIcon : FontIcon
```

Comment:
pay no attention to this `TextBox`/`ProgressRing`/`Slider`, it's just a `FontIcon`, move along 🤣

Reply:
It ain't pretty.. but we can refactor it later I guess 🫠